string GUI=GUI
string ROV=ROV
string JOYSUB=joystick_subscriber
string JOYPUB=joystick_publisher
string JOYCHK=joystick_checker
string CAM0=Cam0/webcam/webcam_stream
string CAM1=Cam1/webcam/webcam_stream
string CAM2=Cam2/webcam/webcam_stream
